python/ct/proto/certificate.proto

```diff
@@ -58,8 +49,7 @@ message X509Description {
   // SHA256 hash from DER form.
   optional bytes sha256_hash = 10;
 
-  // Deprecated - The code to fill this field in was removed.
-  repeated Observation observations = 11;
+  // Field 11 was Observation, no longer produced.
```

Comment:
Could use the `reserved` feature: https://developers.google.com/protocol-buffers/docs/proto#reserved